app/forms/openid_connect_authorize_form.rb

```diff
@@ -68,9 +69,9 @@ def loa3_requested?
 
   private
 
-  def parse_acr_values(acr_values)
-    return [] if acr_values.blank?
-    acr_values.split(' ').compact & Saml::Idp::Constants::VALID_AUTHN_CONTEXTS
+  def parse_to_values(str, possible_values)
+    return [] if str.blank?
+    str.split(' ').compact & possible_values
```

Comment:
I think `split` separates on empty spaces by default (so this arg not necessary) https://ruby-doc.org/core-2.2.0/String.html#method-i-split